frontend/package.json

```diff
@@ -7,9 +7,10 @@
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
     "@varld/popover": "^0.1.0",
+    "particles-bg": "^2.5.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-modal-login": "^1.4.5",
+    "react-modal-login": "^1.3.3",
```

Comment:
could you correct that by keeping the last version **1.4.5**  or not commiting the package.json